tests_requirements.txt

```diff
@@ -7,4 +7,4 @@ moto==2.2.16
 pandas
 pytest
 PyYAML==6.0
-typeguard
+typeguard==2.12.1
```

Comment:
Get rid of this file (and requirements.txt) altogether if we aren't going to use them.

Reply:
test_requirement.txt is for testing propose and it should be isolate for requirement.txt, because we dont want to mix the test pkgs with project pkg.

Reply:
If we're going to hard code the packages and versions in the pip command line, we should not have either requirements.txt file, just the pip commands.

Reply:
We need to add it only to the PR yaml file to mitigate risk.
We are still use it regular when [merging to main](https://github.com/redhat-performance/benchmark-runner/blob/90483db157edbcee2efd82ccb36415cea3a33895/.github/workflows/Perf_Env_Build_Test_CI.yml#L35-L36)